bindings/rascal/models/IP_generic_md.py

```diff
@@ -0,0 +1,103 @@
+import ase.io
+import ase.units
+import numpy as np
+
+from ..utils import BaseIO, load_obj
+from ..neighbourlist.structure_manager import AtomsList, unpack_ase
+
+
+class GenericMDCalculator(BaseIO):
+
+    """Generic MD driver for a librascal model
+
+    Initialize with model JSON and a structure template, and calculate
+    energies and forces based on position/cell updates _assuming the
+    order and identity of atoms does not change_.
+    """
+
+    def __init__(self, model_json, structure_template, assume_pbc=True):
+        """Initialize a model and structure template
+
+        Parameters
+        ----------
+        model_json  Filename for a JSON file defining the potential
+        structure_template
+                    Filename for an ASE-compatible Atoms object, used
+                    only to initialize atom types and numbers
+        assume_pbc  Force the PBC to True (to avoid subtle issues that
+                    sometimes arise when no cell is defined), default True
```

Comment:
This feels a lot like a hack. What kind of issue arise when no cell is defined? And could we work around these issues automatically?

Reply:
This is the automatic workaround. The problem is that ASE sets the cell to be non-periodic if you read a file with no (or unreadable) cell information, _even if you explicitly set a cell later_, so you have to remember to set PBC explicitly as well. I've lost weeks of work because of this misunderstanding.

Although probably the issue will be partially addressed in #323 ; let's discuss it more there.